app/components/Header/test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+
+import { Header, Props } from '.'
+
+const props: Props = {
+  film: {
+    id: 'id',
+    characters: [],
+    comments: [],
+    description: 'description',
+    image: 'image',
+    movie_banner: 'movie_banner',
+    original_title: 'original_title',
+    people: ['PersonName'],
+    release_date: 'release_date',
+    rt_score: '75',
+    running_time: 'running_time',
+    title: 'title',
+  },
+}
+
+describe('<Header />', () => {
+  it(`should render correctly`, () => {
+    const { container } = render(<Header {...props} />, {
+      wrapper: MemoryRouter,
+    })
```

Comment:
```suggestion
    render(<Header {...props} />)
```

Só precisa do `MemoryRouter` quando tiver algum link com rota do `react-router-dom`